src/types/docs-schemas/definitions.schema.json

```diff
@@ -733,7 +733,7 @@
       "title": "AbiType",
       "anyOf": [
         { "title": "StringABI", "type": "string" },
-        { "title": "ArrayABI", "type": "array" }
+        { "title": "ArrayABI", "type": "array", "items": { "type": "string" } }
```

Comment:
Can you confirm that this will work with using JSON abi? 

I.e: 
```
abi: ${file(../abis/Comet.json)}
```
Where Comet.json is given as per https://etherscan.io/address/0x9655ed692f3949288dc217b9795bd543bf88d74f#code

n.b. that ABI contains struct inputs (nested json objects)

Reply:
Just tested and it worked